src/main/java/org/sagebionetworks/bridge/hibernate/HibernateSurveyElement.java

```diff
@@ -0,0 +1,114 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.sagebionetworks.bridge.models.surveys.SurveyElement;
+import org.sagebionetworks.bridge.models.surveys.SurveyRule;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableList;
+
+@Entity
+@Table(name = "SurveyElements")
+public class HibernateSurveyElement implements SurveyElement {
+    @Id
+    private String guid;
+    private String surveyGuid;
```

Comment:
Which reminds me, SurveyElement needs to have createdOn in addition to surveyGuid.